web/src/_services/user_service.js

```diff
@@ -0,0 +1,60 @@
+import config from 'config';
+import { authHeader } from '../_helpers';
+
+export const userService = {
+    login,
+    logout,
+	getAll
+};
+
+function login(username, password) {
```

Comment:
We agreed on using only google oauth2 authentication, or am I mistaken? Because this logic handles username/password authentication (with noneistent api endpoint)